gybitg/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -0,0 +1,398 @@
+                        .IsUnique()
+                        .HasName("UserNameIndex")
+                        .HasFilter("[NormalizedUserName] IS NOT NULL");
+
+                    b.ToTable("AspNetUsers");
+                });
+
+            modelBuilder.Entity("gybitg.Models.AthleteProfile", b =>
+                {
+                    b.Property<string>("UserId")
+                        .ValueGeneratedOnAdd();
+
+                    b.Property<string>("AAUId");
+
+                    b.Property<DateTime>("DateOfBirth");
+
+                    b.Property<DateTime>("HSGraduationDate");
+
+                    b.Property<decimal>("Height");
+
+                    b.Property<string>("HighschoolCoach");
+
+                    b.Property<string>("HighschoolName");
+
+                    b.Property<string>("PersnalBio");
+
+                    b.Property<decimal>("Weight");
+
+                    b.HasKey("UserId");
+
+                    b.ToTable("AthleteProfiles");
+                });
+
+            modelBuilder.Entity("gybitg.Models.AthleteStats", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd();
+
+                    b.Property<decimal>("APG");
+
+                    b.Property<DateTime>("DateOFEntry");
+
+                    b.Property<decimal>("FGAG");
+
+                    b.Property<decimal>("FGG");
+
+                    b.Property<decimal>("FGMG");
+
+                    b.Property<decimal>("FTAG");
+
+                    b.Property<decimal>("FTMG");
+
+                    b.Property<decimal>("FTP");
+
+                    b.Property<int>("GP");
+
+                    b.Property<int>("GS");
+
+                    b.Property<decimal>("MPG");
+
+                    b.Property<decimal>("PPG");
+
+                    b.Property<decimal>("RPG");
+
+                    b.Property<decimal>("TPAG");
+
+                    b.Property<decimal>("TPMG");
+
+                    b.Property<decimal>("TPP");
+
+                    b.Property<string>("UserId")
+                        .IsRequired();
+
+                    b.HasKey("Id");
+
+                    b.ToTable("AthleteStats");
+                });
+
+            modelBuilder.Entity("gybitg.Models.CoachAthlete", b =>
+                {
+                    b.Property<string>("CoachId");
+
+                    b.Property<string>("AthleteId");
+
+                    b.HasKey("CoachId", "AthleteId");
+
+                    b.HasIndex("AthleteId");
+
+                    b.ToTable("CoachAthletes");
+                });
+
+            modelBuilder.Entity("gybitg.Models.CoachProfile", b =>
+                {
+                    b.Property<string>("UserId")
+                        .ValueGeneratedOnAdd();
+
+                    b.Property<string>("AAUId");
+
+                    b.Property<string>("Achievments");
+
+                    b.Property<int>("Lossess");
+
+                    b.Property<string>("PersnalBio");
```

Comment:
very small point:  PersnalBio is misspelled.  Should be PersonalBio.  It must have been originally misspelled in the CoachProfile model, that's why Intellisense misspelled it for you.